From 6eac0c78ebcc764508e43ae5f492cb8693305b52 Mon Sep 17 00:00:00 2001
From: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
Date: Fri, 8 May 2015 14:23:09 +0200
Subject: [PATCH] touchscreen: rafi: Fix event report

Don't separate key type BTN_TOUCH from ABS type, since this is one packet.
Otherwise the events are misinterpreted.

Signed-off-by: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
---
 drivers/input/touchscreen/rafi_ts_input.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/rafi_ts_input.c b/drivers/input/touchscreen/rafi_ts_input.c
index cdef808..6325add 100644
--- a/drivers/input/touchscreen/rafi_ts_input.c
+++ b/drivers/input/touchscreen/rafi_ts_input.c
@@ -70,7 +70,8 @@ static void
 emit_button_down(struct rafi_ts_input * thiz)
 {
     input_report_key(thiz->input_dev, BTN_TOUCH, 1);
-    input_sync(thiz->input_dev);
+// Don't separate BTN_TOUCH 1 from ABS_X/Y
+//    input_sync(thiz->input_dev);
 }
 
 
-- 
2.1.4

